src/features/FormPage/OrderForm/OrderForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+
+const OrderForm = () => {
+
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [firstLineAddress, setFirstLineAddress] = useState('');
+    const [postcode, setPostcode] = useState('');
+    const [orderId, setOrderId] = useState('');
+
+    const formData = {
+        'name': name,
+        'firstLineOfAddress': firstLineAddress,
+        'postcode': postcode,
+        'email': email
+    }
+
+    // const resetForm = () => {
+    //     setName('')
+    //     setEmail('')
+    //     setFirstLineAddress('')
+    //     setPostcode('')
+    // }
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        fetch('http://localhost:3001/orders', {
+                "method": "POST",
+                "body": JSON.stringify(formData),
+                "headers":
+                    {
+                        "content-type": "application/JSON"
+                    }
+            }
+        )
+            .then (res => res.json())
+            .then ((data) => {
+                //do stuff with your data
+                localStorage.setItem("orderId", data._id)
```

Comment:
Probably worth double-checking that you actually got an _id back before you put it into localStorage.

Also worth adding a basic error message if you get any validation problems, ie if a postcode is malformed